lib/trento_web/controllers/v1/host_controller.ex

```diff
@@ -28,15 +28,35 @@ defmodule TrentoWeb.V1.HostController do
     render(conn, "hosts.json", hosts: hosts)
   end
 
-  operation :heartbeat, false
+  operation :heartbeat,
+    summary: "Update the heartbeat for a host",
+    tags: ["Agent"],
+    description:
+      "Update the heartbeat for a host. This is used by the agents to signal that they are still alive.",
+    parameters: [
+      id: [
+        in: :path,
+        required: true,
+        type: %OpenApiSpex.Schema{type: :string, format: :uuid}
+      ]
+    ],
+    responses: [
+      accepted: "The heartbeat has been updated",
+      not_found: TrentoWeb.OpenApi.Schema.NotFound.response(),
+      bad_request: TrentoWeb.OpenApi.Schema.BadRequest.response(),
```

Comment:
Currently this operation returns either 204, instead of 202 and never returns a 400 nor a 422.
Is there any behaviour that I missing that takes care of 400s/422s?

However if the plan is to add that case and we want to have that documented in the meantime, ignore the comment.

Reply:
The 202 is indeed wrong.

422 and 400 are handled by `OpenApiSpex cast_and_validate` plug and Phoenix respectively.